contracts/button/lib.rs

```diff
@@ -47,6 +47,7 @@ mod button_game {
         #[ink(topic)]
         by: AccountId,
         when: BlockNumber,
+        score: Balance,
```

Comment:
should we make it `#[ink(topic)]`?

Reply:
nah I doubt it's interesting to index by scores, they will be all over the place no?

Reply:
I was misled by 'needed for indexing purposes' :P